examples/testapp/src/utils/cleanupSDKLocalStorage.ts

```diff
@@ -0,0 +1,8 @@
+export const cleanupSDKLocalStorage = () => {
+  // remove all keys that contains 'cbw' or 'CBW'
+  Object.keys(localStorage).forEach((key) => {
+    if (key.startsWith('cbwsdk.') || key.startsWith('-CBWSDK:')) {
```

Comment:
Nice. wonder if we should clean up storage values for apps now that we've moved storage to zustand

Reply:
can you elaborate?

Reply:
SDK side when we create a new instance of the SDK should we clear all storage with the `-CBWSDK` prefix

Reply:
ahh we should ya, but that should be a SDK function, let's do it in a follow PR?

Reply:
actually no, i take it back, walletlink module is still using -CBWSDK prefix, so we will need to wait until WL is gone

Reply:
Ahh thats right, thats why I didnt on the storage pr. I forgot already lol